### DISCUSSION TITLE
Where is the Config Data stored

### DISCUSSION THREAD

**Discussion by Alex-joomla** [dougiteixeira/proxmoxve#279] [Q&A] [answered]
Hi, I changed the IP of my PVE Host, how can I change it in the integration or where is this data stored where i can change it for example with a text editor.

Regards

**Comment by dougiteixeira** [comment]
Sorry for the delay... the settings are stored in the .storage directory, but it is not recommended to change them manually.

Next month I will add the ability to reconfigure the integration without having to remove it.

**Comment by psyrius1** [comment]
Hi, I have the same issue. I changed my PVE host ip address. I think we still don't have the possibility to reconfigure. Where can i change the settings manually?

**Reply by shaftspanner** [reply]
Did you manage to solve this?  I've just had to rebuild my network and I've got the same issue - my PVE host ip address has changed but I can't find anything in the .storage directory that references it

**Comment by dougiteixeira** [comment, marked as answer]
Implemented in #493
